esp/scm/ws_config2.ecm

```diff
@@ -20,7 +20,7 @@ ESPstruct StatusMsgType
 {
     string msgLevel;                // message level ("info", "warning", "error", "fatal")
     string nodeId;                  // node identifier that generated this status message
-    string nodeName("");            // node name (not unique, use nodeID to retrieve node)
```

Comment:
I thought the field names had been capitalized. The standard as far as I know is to capitalize the field names.

Reply:
Changed to capitalize first letter of each member name.